src/plugin.ts

```diff
@@ -4,18 +4,36 @@ import { createClient } from "@supabase/supabase-js";
 import { LogReturn, Logs } from "@ubiquity-os/ubiquity-os-logger";
 import { createAdapters } from "./adapters";
 import { userPullRequest, userSelfAssign, userStartStop, userUnassigned } from "./handlers/user-start-stop";
-import { Context, Env, PluginInputs } from "./types";
+import { Context, Env, GitHubIssueSearch, PluginInputs } from "./types";
 import { addCommentToIssue } from "./utils/issue";
 
+async function listOrganizations(): Promise<string[]> {
+  const res = await fetch("https://raw.githubusercontent.com/ubiquity/devpool-directory/refs/heads/__STORAGE__/devpool-issues.json");
```

Comment:
I wish we had a mechanism to detect if this breaks to respond quickly because that system may be changed without notice in the future. I suppose for now you should throw a specific and descriptive error if it 404.